GameData/RP-1/Contracts/Earth Observation 1/EOSCherenkov.cfg

```diff
@@ -73,23 +73,14 @@ CONTRACT_TYPE
 			type = Sequence
 			title = Survive in orbit for a day and transmit science
 
-			PARAMETER
-			{

```

Comment:
Sorry, it's been a bit. Why are the EC requirements being removed from these?

Reply:
My logic was that since we have the science requirement, they'll drive needing EC